data/pins_sonata.xdc

```diff
@@ -19,8 +19,8 @@ set_property -dict { PACKAGE_PIN A11 IOSTANDARD LVCMOS33 } [get_ports {led_user[
 set_property -dict { PACKAGE_PIN F13 IOSTANDARD LVCMOS33 } [get_ports {led_user[6]}];
 set_property -dict { PACKAGE_PIN F14 IOSTANDARD LVCMOS33 } [get_ports {led_user[7]}];
 
-## JTAG test
-set_property -dict { PACKAGE_PIN H17 IOSTANDARD LVCMOS33 } [get_ports tck_i];
+## User JTAG
+set_property -dict { PACKAGE_PIN E15 IOSTANDARD LVCMOS33 } [get_ports tck_i];
```

Comment:
Suggestion:
```
## USR JTAG - rev 0.3+
set_property -dict { PACKAGE_PIN E15 IOSTANDARD LVCMOS33 } [get_ports tck_i];
## USR JTAG - rev <= 0.2
# set_property -dict { PACKAGE_PIN H17 IOSTANDARD LVCMOS33 } [get_ports tck_i];
```

I think at this stage it's worth leaving the earlier pinout in the xdc file for a bit, because we still have few boards around and may yet find ourselves trying to draw comparisons against earlier boards.
For more esoteric stuff such as RPi I2C pins it's probably not an issue, but for the RST button and the USR_JTAG it's more critical.

Reply:
Thanks, this is now done.